src/common/actions/snap-builds.js

```diff
@@ -18,27 +20,38 @@ const REQUEST_POST_OPTIONS = {
   credentials: 'same-origin'
 };
 
-export function fetchBuildsSuccess(builds) {
+export function fetchBuildsSuccess(id, builds) {
   return {
     type: FETCH_BUILDS_SUCCESS,
-    payload: builds
+    payload: {
+      id,
+      builds
+    }
   };
 }
 
-export function fetchBuildsError(error) {
+export function fetchBuildsError(id, error) {
```

Comment:
For now it's the simplest way to go.

We will likely fetch this in parallel for the enabled repositories, so it's easier to just keep whole status separately for each repo, so they don't affect each other. For example error fetching build status of one repo doesn't mean we can't have successfully fetched build status for other repo and show it in UI.

I'm fine with discussing how we further go with storing such data, but what we need right now this seems enough.